packages/ubuntu_provision/test/network/network_page_test.dart

```diff
@@ -72,6 +74,7 @@ void main() {
   });
 
   testWidgets('selects wifi', (tester) async {
+    await tester.binding.setSurfaceSize(const Size(800, 800));
```

Comment:
Could we add a TODO here and/or on the network page to keep track of the places that will need some polishing?

Reply:
Done!